microsoftgraph/microsoft-graph-explorer-v4
# Fix: enter key runs previous query

## Overview

Fixes #2338 

### Notes

It fails for the use case where a user types in something and it is the only thing in the suggestions. Since we didn't assume this becomes the selected suggestion, the query doesn't get changed.

It is fixed by appending the suggestion to the URL

## Testing Instructions

* Run `/me`
* Change `me` to `users`
* Hit enter
* Notice the results for `/users` in the response
